DOC: Update installation part in readme (#253)
xorbitsai/inference: README.md

## Getting Started
Xinference can be installed via pip from PyPI. It is highly recommended to create a new virtual
environment to avoid conflicts.

### Installation
```bash
$ pip install "xinference"
```
`xinference` installs basic packages for serving models. 

#### Installation with GGML

> Serving models with ggml format, you need to install some extra dependencies as following:

Reviewer: ```suggestion
To serve ggml models, you need to install the following extra dependencies:
```

```bash
$ pip install "xinference[ggml]"
```
If you want to achieve acceleration on 
different hardware, refer to the installation documentation of the corresponding package.
- [llama-cpp-python](https://github.com/abetlen/llama-cpp-python#installation-from-pypi-recommended) is required to run `baichuan`, `wizardlm-v1.0`, `vicuna-v1.3` and `orca`.
- [chatglm-cpp-python](https://github.com/li-plus/chatglm.cpp#getting-started) is required to run `chatglm` and `chatglm2`.

#### Installation with PyTorch

> Serving models with PyTorch, you need to install some extra dependencies as following:

Reviewer: ```suggestion
To serve PyTorch models, you need to install the following extra dependencies:
```

```bash
$ pip install "xinference[pytorch]"
```

#### Installation with all dependencies

> If you want to serve all supported models, install all dependencies:

Reviewer: ```suggestion
If you want to serve all the supported models, install all the dependencies:
```
